tests/unit/test_eurostat.py

```diff
@@ -1,211 +1,77 @@
-        with caplog.at_level(logging.ERROR):
-            eurostat.validate_params(
-                dataset_code=eurostat.dataset_code, url=URL, params=params
-            )
-    assert (
-        "Parameters codes: 'total1 | non_existing_code' are not available. "
-        "Please check your spelling!" in caplog.text
-    )
-
-
-def test_parameter_codes_as_list_logger(caplog):
-    """Tests error logging for incorrect parameter codes structure."""
-    params = {"hhtyp": ["totale", "nottotale"], "indic_il": "med_e"}
-    eurostat = EurostatMock(dataset_code="ILC_DI04", params=params)
-
-    with pytest.raises(TypeError, match="Wrong structure of params!"):  # noqa: SIM117
-        with caplog.at_level(logging.ERROR):
-            eurostat.validate_params(
-                dataset_code=eurostat.dataset_code, url=URL, params=params
-            )
-    assert (
-        "You can provide only one code per one parameter as 'str' in params! "
-        "CORRECT: params = {'unit': 'EUR'} | INCORRECT: params = "
-        "{'unit': ['EUR', 'USD', 'PLN']}" in caplog.text
-    )
-
-
-def test_wrong_parameters(caplog):
-    """Tests error logging for incorrect parameter keys."""
-    params = {"hhhtyp": "total", "indic_ilx": "med_e"}
-    eurostat = EurostatMock(dataset_code="ILC_DI04", params=params)
-
-    with pytest.raises(ValueError, match="Wrong parameters or codes were provided!"):  # noqa: SIM117
-        with caplog.at_level(logging.ERROR):
-            eurostat.validate_params(
-                dataset_code=eurostat.dataset_code, url=URL, params=params
-            )
-    assert (
-        "Parameters: 'hhhtyp | indic_ilx' are not in dataset. "
-        "Please check your spelling!" in caplog.text
-    )
-
-
-def test_correct_params_and_dataset_code(caplog):
-    """Tests that the data retrieval feature returns a non-empty DataFrame.
-
-    For valid dataset code.
-    """
-    eurostat = EurostatMock(
-        dataset_code="ILC_DI04", params={"hhtyp": "total", "indic_il": "med_e"}
-    )
-    df = eurostat.to_df()
-
-    assert isinstance(df, pd.DataFrame)
-    assert not df.empty
-    assert caplog.text == ""
-
-
-def test_wrong_needed_columns_names(caplog):
-    """Tests error logging for incorrect names of requested columns."""
-    eurostat = EurostatMock(
+    eurostat = Eurostat(
         dataset_code="ILC_DI04",
         params={"hhtyp": "total", "indic_il": "med_e"},
-        columns=["updated1", "geo1", "indicator1"],
+        columns=["geo", "time"],
     )
 
-    with pytest.raises(ValueError, match="Provided columns are not available!"):  # noqa: SIM117
-        with caplog.at_level(logging.ERROR):
-            eurostat.to_df()
-    assert (
-        "Name of the columns: 'updated1 | geo1 | indicator1' are not in DataFrame. "
-        "Please check spelling!" in caplog.text
-    )
+    df = eurostat.to_df()
 
+    assert df.shape == (850, 7)
```

Comment:
That doesn't make sense, since mock data is like 15 lines long, so something is wrong with the patching.